### ISSUE TITLE
Website Squatted

### ISSUE THREAD

**Issue by destructobeam** [rack-app/rack-app#49]
It looks like the registration for the website you have listed for the project expired and has been taken over.

**Comment by destructobeam** [comment]
Just realised I already made an issue for this on the website repo. Apologies.

**Comment by adamluzsi** [comment]
No worries. I primarily watch this project for issues, so you posted this in the perfect place.  👍

**Comment by adamluzsi** [comment]
Oh, Heroku free tier is over... 🙈

**Comment by adamluzsi** [comment]
Do you have any suggestions on where I should move the content @destructobeam? 
Converting everything into MD and moving it under the GH wiki page perhaps?

**Comment by destructobeam** [comment]
You can use GitHub pages with Jekyll. That might be easiest as that’s also Ruby?

**Comment by adamluzsi** [comment]
I will look into that; thank you for the great tip!

**Comment by waghanza** [comment]
or use fly.io @adamluzsi (and maybe ping me we http://www.rack-app.com/ will be up)

**Comment by adamluzsi** [comment]
Hey there, @waghanza!

I had a chance to check out fly.io, and it seems like a great platform!
But here's the thing: it might be too much for our needs when I think about it. 
Our primary focus is organizing examples for the framework, and since my website mainly consists of Gists from GitHub, it seems more practical to stick with GitHub only.
Writing a wiki might take a bit longer, but it'll be easier for most users to access the examples in the long run.

Work has been keeping me busy lately, so I could not start rewriting the documentation as a wiki.
But I'm excited to dive in, and I appreciate your understanding!
Feel free to let me know if you have any further thoughts or suggestions. 
In the meantime, if you need the website today, please try this:

``` shell
git clone git@github.com:rack-app/website.git rack-app-website
cd rack-app-website
bundle 
bundle exec rackup
```

**Comment by waghanza** [comment]
I understand that https://fly.io/ could be more than needed, but probably then https://readthedocs.org/ could be cool too

I'm not keen on letting user build the doc himself/herself.

PS : And also benchmarking section with `ruby 2.3.0p0` is a, imho, not a great promotion made for `rack-app`. I'll plan to make a graphql endpoint for my benchmarking project if you want

**Comment by adamluzsi** [comment]
Hey, there @waghanza,

When my daughter was born, everything shifted for me, and naturally, family and essentials took precedence over projects like rack-app. That said, I'm quite proud of how well `rack-app` has been designed. It's held up without any significant issues (barring the website hiccup), so upkeep has been manageable. I recognise that lately, it might seem like the project has taken a backseat. Most of my energy went into the essentials: maintenance and responding to support queries. The extras, well, they sort of got moved to the back burner.

On the brighter side, a recent change in my employment situation has given me some unexpected free time. Once I've secured a new role, I hope to use this extra time to revisit some long-standing plans with `rack-app`, including refreshing the documentation, updating the benchmark, and improving the startup time.

**Comment by waghanza** [comment]
Congrats for your parenthood @adamluzsi  🎉

**Comment by destructobeam** [comment]
@adamluzsi Congrats! That aside, the domain name has been grabbed by someone else. So might be worth just removing the link to the squatted site in the description until a new site/domain is up?

**Comment by adamluzsi** [comment]
Good point. I do some late night work today and create a wiki and remove the domain link.

**Comment by adamluzsi** [comment]
Hello @destructobeam, I made a wiki page today, can you give a quick look if it would be a sufficient replacement for the website?

**Comment by destructobeam** [comment]
@adamluzsi Sorry for the very late reply. It looks good, there are just some references to the original website in the READEME that need updating to point to the wiki. Don't know if I'll get any time, but if I do I will create a pull request. Should be a quick find and replace.

Thanks!

**Comment by adamluzsi** [comment]
It would be an honour if you would lend your name to the project with your contribution! 
Then I don't even try to find them. :D
Many thanks for helping out, Simon (@destructobeam)!
